oak/server/storage/storage_processor.cc

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright 2019 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "oak/server/storage/storage_processor.h"
+
+#include "absl/memory/memory.h"
+#include "absl/strings/escaping.h"
+#include "asylo/util/cleansing_types.h"
+#include "asylo/util/logging.h"
+#include "grpcpp/create_channel.h"
+
+namespace oak {
+
+constexpr size_t kMaxMessageSize = 1 << 16;
```

Comment:
put this and `GetStorageId` in an unnamed namespace, so they don't get visible outside?

Reply:
Done.